scripts/e2e-custom.sh

```diff
@@ -156,6 +172,11 @@ test_download() {
         ilab model download --repository instructlab/merlinite-7b-lab
     fi
 
+    if [ "$DEV_PREVIEW_FEATURE_GATING" -eq 1 ]; then
+        step Downloading embedding model
+        ilab model download --repository "${GRANITE_EMBEDDING_MODEL}"
```

Comment:
probably needs `--hf-token "${HF_TOKEN}`

Reply:
Will be handled by https://github.com/instructlab/instructlab/pull/2997 - `ilab` automatically pulls from the env var if it is present